src/pages/login/login.ts

```diff
@@ -195,4 +217,13 @@ export class LoginPage extends BasePageComponent {
       this.loadingSpinner = null;
     }
   }
+
+  private createLog(logType: LogType, actionType: string, message: string): Log {
```

Comment:
Unused param `actionType`. Did you intend on adding this to the log?

Reply:
Great spot. Unintentional. Removing superfluous parameter now.